debugger/debugger.go

```diff
@@ -81,6 +82,7 @@ func (db *Debugger) createServer(addr string) {
 
 func (db *Debugger) pause(w http.ResponseWriter, r *http.Request) {
 	db.emu.TogglePause()
+	db.sendState(w)
```

Comment:
This handles both pause and unpause, maybe only send the state when pausing. when unpausing we don't really know what we are sending

Reply:
:+1: